package.json

```diff
@@ -9,7 +9,9 @@
     "preview": "vite preview",
     "lint": "eslint src --ext '[.ts, .tsx]' --resolve-plugins-relative-to .",
     "lint:fix": "yarn lint --fix",
-    "local": "docker-compose -f ./docker.compose-development.yaml up -d && vite"
+    "local": "docker-compose -f ./docker.compose-development.yaml up -d && vite",
+    "development": "node sdk-package-install.js",
```

Comment:
script name is confusing, also the `.js` filname. can we change it to something that is more descriptive?

Reply:
also change in other repos